src/swap/utils/formatDecimals.ts

```diff
@@ -1,3 +1,5 @@
+import JSBI from 'jsbi';
```

Comment:
Direct imports are strictly not advised as are impossible to tree-shake. This is something the Javascript community  moved away from in 2016, after the big drama libraries like `lodash` created.

I did not checked the library directly, but is it possible to just import what we need, or we always have to get everything?

Importing always everything is quite a risk for us to take, as open source library.

Reply:
not the expert here - but seems like this is a known (unresolved) issue https://github.com/GoogleChromeLabs/jsbi/issues/22

however, performance wise it should be on par with native BigInt implementations from newer versions of browsers

```
On average, JSBI is performance-competitive with the native implementation that Google Chrome is currently shipping.
```

Reply:
Should we just use the native when detected? And default to do nothing when is not available?

It seems the library is pretty heavy

Reply:
migrated to native `BigInt`